examples/lock-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -72,3 +76,19 @@ void AppDeviceCallbacks::OnOnOffPostAttributeChangeCallback(EndpointId endpointI
 exit:
     return;
 }
+
+void AppDeviceCallbacks::OnDoorLockPostAttributeChangeCallback(EndpointId endpointId, AttributeId attributeId, uint8_t * value)
+{
+    VerifyOrExit(attributeId == ZCL_LOCK_STATE_ATTRIBUTE_ID, ESP_LOGI(TAG, "Unhandled Attribute ID: '0x%04x", attributeId));
```

Comment:
Please use the namespaced attribute ids, not the to-be-removed ZCL_* constants.

Reply:
Done